docs/ProductSummaryBuyButton.md

```diff
@@ -1,72 +1,62 @@
+1. Import the Shelf app to your theme's dependencies on the `manifest.json`, for example:
 
 ```json
-  "product-summary-buy-button": {
-    "component": "ProductSummaryBuyButton"
+  dependencies: {
+    "vtex.product-summary": "2.x"
   }
 ```
 
-### Configuration
+2. Add `product-summary-buy-button` anywhere inside the `product-summary.shelf`:
+
+```diff
+ {
+   "product-summary.shelf": {
+       "children": [
+           "product-summary-image",
+           "product-summary-name",
+           "product-rating-inline",
+           "product-summary-space",
+           "product-summary-price",
++          "product-summary-buy-button"
+       ]
+   },
+ }
+```
 
-Through the Storefront, you can change the `ProductSummaryBuyButton`'s behavior and interface. However, you also can make in your theme app.
+### Props
 
 | Prop name           | Type      | Description                                                                                 | Default value         |
 | ------------------- | --------- | ------------------------------------------------------------------------------------------- | --------------------- |
 | `isOneClickBuy`     | `Boolean` | Should redirect to checkout after clicking on buy                                           | `false`               |
 | `buyButtonText`     | `String`  | Custom buy button text                                                                      |                       |
 | `displayBuyButton`  | `Enum`    | Set display mode of buy button (displayButtonAlways, displayButtonHover, displayButtonNone) | `displayButtonAlways` |
 | `customToastURL`  | `String`    | Set the link associated with the Toast created when adding an item to your cart.  | `/checkout/#/cart` |
-| `buyButtonBehavior` | `Enum` | What the buy button should do when you click it, if you pass `default` it will add to cart only if there is only one SKU of that product (default, alwaysGoToProduct) | `default`
+| `buyButtonBehavior` | `Enum` | What will happen when the user clicks on the button. Check the options bellow. | `default`
 
-### Styles API
+- Possible values for `buyButtonBehavior`:
 
-This app provides some CSS classes as an API for style customization.
+| Value | Description |
+| --- | --- |
+| `default` | It will add to cart if the product has one SKU. It will link to the product page if it has more than one SKU. |
+| `alwaysGoToProduct` | It will always link to the product page. |
+| `alwaysAddToCart` | It will always add the selected item to cart. Be careful: use this option only if the user is able to select all variations of the product. Example: if the product has Color and Size, you should show the SKU Selector of both variations so the user can select the color and size they want, otherwise the button will add a product that the user was not able to choose, for example, the first Size option. |
```

Comment:
```suggestion
| `alwaysAddToCart` | It will always add the selected item to cart. **Be careful**: use this option only if the user is able to select all variations of the product. Example: if the product has Color and Size, you should show the SKU Selector of both variations so the user can select the color and size they want, otherwise the button will add a product that the user was not able to choose, for example, the first Size option. |
```